watertap/unit_models/electrodialysis_0D.py

```diff
@@ -205,7 +205,7 @@ def build(self):
 
         self.cell_pair_num = Var(
             initialize=1,
-            domain=NonNegativeIntegers,
+            domain=NonNegativeReals,
```

Comment:
Do we wanna change this now or leave it when optimization is actually needed?  We briefly mentioned that there might be a way to walk around this var when doing optimization by optimizing over the total membrane size.  So maybe we can keep this constrain without hurting any goals.  But this change should not impact functions of the unit model so far.

Reply:
I feel like we might as well change this now. It won't have any impact on the model or how the model solves, but would allow a user to try to optimize for the cell numbers if they choose to do so.